sys-libs/libcxxabi/libcxxabi-3.9.0.ebuild

```diff
@@ -72,6 +72,11 @@ multilib_src_configure() {
 			-DLIT_COMMAND="${EPREFIX}"/usr/bin/lit
 		)
 	fi
+	if use elibc_musl; then
+		mycmakeargs+=(
+			-DCMAKE_CXX_FLAGS="-D_LIBCPP_HAS_MUSL_LIBC=1"
```

Comment:
Ok, I get that. So how about putting it in CPPFLAGS? Could you try that? Also, please add elibc_musl to IUSE, I prefer having all used flags listed.